docs/fundamentals/toc.yml

```diff
@@ -2316,7 +2316,7 @@ items:
         href: ../core/porting/upgrade-assistant-wpf-framework.md
       - name: Windows Forms
         href: ../core/porting/upgrade-assistant-winforms-framework.md
-      - name: ASP.NET Core
+      - name: ASP.NET
```

Comment:
The parent toc node says: "Port from .NET Framework".
So I was expect the "Core" suffix is there to mean this is migration **to** (not from) ASP.NET Core, which is correct.

Reply:
I'm still redoing the entire migration section, but no, the articles are focused on migrating a **type of project** to .NET 5.